### DISCUSSION TITLE
How do you write a compressed GeoTiff

### DISCUSSION THREAD

**Discussion by JonesRusty** [corteva/rioxarray#327] [Q&A] [answered]
Hi,

It is probably really easy but I don't see how to write a compressed TIF file. I would want to use a lossless compression algorithm like L77 or LZW.

Can anyone point me in the right direction?

**Comment by snowman2** [comment, marked as answer]
The good news is that it is pretty simple. You just pass the `compress` kwarg into `to_raster`:

```python
xds.rio.to_raster("file.tif", compress='LZW')
```

**Reply by JonesRusty** [reply]
Thanks Alan. It is great to post a message to the internet and get an answer from a local and familiar face.

**Reply by snowman2** [reply]
Happy to help!
